homu/main.py

```diff
@@ -439,9 +444,21 @@ def fetch_mergeability(mergeable_que):
         finally:
             mergeable_que.task_done()
 
+def arguments():
+    parser = argparse.ArgumentParser(description = 'A bot that integrates with GitHub and '
+                                     + 'your favorite continuous integration service')
+    parser.add_argument('-v', '--verbose',
+                        action='store_true', help='Enable more verbose logging')
+
+    return parser
```

Comment:
I think `arguments()` should return parsed arguments, not a parser. We may call `.parse_args()` before returning here.